doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -946,6 +982,55 @@ root@sonic:/home/admin# show interfaces status
 -----------  -------  -------  -----  -----  -----------  ------  ------  -------  ------  ----------
   Ethernet0  0,1,2,3     200G   9100    N/A  Ethernet1/1  routed      up       up     N/A         N/A
 ```
+### 3.5 Console Management
+The console access to smartswitch needs to support DPU access in addition to the switch.
+* By default the management port should be connected to the switch CPU console
+* Provide a toggle option to access the BMC console in order to recover the CPU in case of a failure
+* Once inside the switch, the DPUs should be accesssible from it with an utility. Platforms can use this utility to warp the underlying generic utilitites such as picocom or rconsole.
+    * Ex: xconsole -b <baud-rate> -m <module-name>
+* The default baud rate should be 9600
```

Comment:
As this is a serial console from NTP to the DPU there should not be restrictions for the baud rate. The baud rate can remain up to the vendor.

Reply:
The baud rate is programmable, but Microsoft has requested the default baud rate to be set to 9600.  So I called out the default baud rate.

Reply:
If Price is ok removing the default baud rate I can do it.

Reply:
@prgeor do we need to limit the baud rate? The serial connection between NPU and DPU is internal to the system, so I assume any baud rate is good while both sides are aligned.

Reply:
We don't have to. My understanding is that the existing data centers have all their equipments connecting at 9600.  So, setting the default of the smartswitch baud rate to 9600 will not require any additional script change at the datacenter end. I tried my best to clarify your question.  However, I removed that line as well :)